ginivision/src/main/java/net/gini/android/vision/analysis/AnalysisFragmentImpl.java

```diff
@@ -394,6 +410,7 @@ private AnalysisHint getNextHint() {
 
     void onStop() {
         mStopped = true;
+        mHandler.removeCallbacks(mHintStartRunnable);
```

Comment:
Remove the delayed runnable for showing the hints. This prevented the activity from being garbage collected for the duration of the `HINT_START_DELAY`.